husky_directory/templates/index.html

```diff
@@ -51,22 +51,22 @@
 <!--              ga('send', 'pageview');-->
 <!--        </script>-->
 
-        <script type="text/javascript">
-        // clear out the global search input text field
-        $(document).ready(function() {
-            $('#searchInput').focus(function() {
-                var $this = $(this)
-                if ($this.val() == $this.prop('defaultValue'))
-                    $this.val('')
-
-            }).blur(function() {
-                var $this = $(this)
-                if ($this.val() == '')
-                    $this.val($this.prop('defaultValue'))
-            });
-
-        })
-        </script>
+<!--        <script type="text/javascript">-->
+<!--        // clear out the global search input text field-->
+<!--        $(document).ready(function() {-->
+<!--            $('#searchInput').focus(function() {-->
+<!--                var $this = $(this)-->
+<!--                if ($this.val() == $this.prop('defaultValue'))-->
+<!--                    $this.val('')-->
+
+<!--            }).blur(function() {-->
+<!--                var $this = $(this)-->
+<!--                if ($this.val() == '')-->
+<!--                    $this.val($this.prop('defaultValue'))-->
+<!--            });-->
+
+<!--        })-->
+<!--        </script>-->
```

Comment:
Can this just be deleted instead of commented out? Otherwise, can you include a comment explaining why it needs to stay there?